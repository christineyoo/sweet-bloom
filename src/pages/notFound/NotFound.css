#not-found {
  margin: 2rem;
}

#not-found-h1 {
  font-size: 2rem;
  margin-bottom: 20rem;
}

@media all and (min-width:800px) {
  #not-found {
    margin: 5rem;
  }

  #not-found-h1 {
    font-size: 3rem;
    margin-bottom: 35rem;
  }
}