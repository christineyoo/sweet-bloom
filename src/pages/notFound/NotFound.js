import React, { Component } from 'react';

class NotFound extends Component {
  render() {
    return (
      <h1 style={{ 'font-size': '5rem' }}>
        Sorry, the page you are looking for is not here...
      </h1>
    );
  }
}

export default NotFound;
