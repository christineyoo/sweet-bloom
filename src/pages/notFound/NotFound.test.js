import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './NotFound';

it('renders NotFound UI as expected', () => {
  const tree = renderer.create(<NotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});
