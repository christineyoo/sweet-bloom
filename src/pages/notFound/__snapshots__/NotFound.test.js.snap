// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NotFound UI as expected 1`] = `
<div
  id="not-found"
>
  <h1
    id="not-found-h1"
  >
    Sorry, the page you are looking for is not here...
  </h1>
</div>
`;
